mne_bids/inspect.py

```diff
@@ -0,0 +1,128 @@
+from pathlib import Path
+import matplotlib
+import matplotlib.pyplot as plt
+from matplotlib.widgets import Button
+
+from mne_bids import read_raw_bids, mark_bad_channels
+from mne_bids.read import _from_tsv
+
+matplotlib.use('Qt5Agg')  # XXX Discuss!
```

Comment:
I would not do this. First I don't think it's mandatory. @cbrnr will tell you that it will work with macos backend :) and mne-bids will be working without matplotlib installed. matplotlib is not a mandatory dep of mne-python and it should not be for mne-bids

Reply:
> it will work with macos backend :)

Ok I can drop this line that uses the PyQt backend.



> mne-bids will be working without matplotlib installed. matplotlib is not a mandatory dep of mne-python and it should not be for mne-bids

Well that's all nice and such, but the inspector simply will not work without Matplotlib, there's no way around it. Either you have it, or you'll get an `ImportError`…

Reply:
I've now changed the backend selection such that it will only request the PyQt backend if it's installed, and otherwise use the default. Good?

3bba9cb

Reply:
I will rework the runtime dependency stuff shortly.

Reply:
So apparently I'm already the macOS backend guy 🤣 - but yes, it does work and there's no need to use PyQt5!